LFS171x/docs/introduction-to-hyperledger-fabric.md

```diff
@@ -420,7 +420,7 @@ $ curl -sSL https://goo.gl/6wtTN5 | bash -s 1.2.0
 
 __**NOTE:**__ Check [http://hyperledger-fabric.readthedocs.io/en/release-1.1/samples.html#binaries](http://hyperledger-fabric.readthedocs.io/en/release-1.1/samples.html#binaries) for the latest URL (the 'https://goo.gl/6wtTN5' in the above curl command) to pull in binaries.
 
-This command downloads binaries for **cryptogen**, **configtxgen**, **configxlator**, **peer** AND downloads the Hyperledger Fabric Docker images. These assets are placed in a **bin** subdirectory of the current working directory.
+This command downloads binaries for **cryptogen**, **configtxgen**, **configxlator**, **peer** AND downloads the Hyperledger Fabric Docker images. These assets are placed in the **fabric-samples/bin** subdirectory of the current working directory.
```

Comment:
The documentation says `places them in the bin sub-directory of the current working directory.`

Reply:
I started with an empty directory and called the command for the 1.3 version and I am still seeing the same behavior, where these binaries are appearing in fabric-samples/bin.  Perhaps the reason is related to this error message that shows up right at the beginning, but doesn't seem to otherwise impact the rest of the example running "bash: line 181: [: too many arguments".

nathalie@nathalie-HP-EliteBook-8460p:~/Hyperledger/LFS171x/Fabric$ curl -sSL http://bit.ly/2ysbOFE | bash -s 1.3.0
bash: line 181: [: too many arguments
...

Once everthing has finished running from that curl command above, an ls shows that the only thing in the current working directory is "fabric-samples"

nathalie@nathalie-HP-EliteBook-8460p:~/Hyperledger/LFS171x/Fabric$ ls
fabric-samples

I have to look in the bin subdirectory of fabric-samples:
nathalie@nathalie-HP-EliteBook-8460p:~/Hyperledger/LFS171x/Fabric$ ls fabric-samples/bin
configtxgen	cryptogen  fabric-ca-client  	idemixgen  peer
configtxlator  discover   get-docker-images.sh  orderer

This is on a Linux box running Ubuntu 16.04.